test/integration_test_helper.rb

```diff
@@ -15,9 +15,20 @@ class IntegrationTest < Krane::TestCase
     end
 
     def run
-      super { @namespace = TestProvisioner.claim_namespace(name) }
+      super do
+        @namespace = TestProvisioner.claim_namespace(name)
+        @global_fixtures_deployed = []
```

Comment:
Perhaps `global_fixtures_paths_deployed`  it wasn't clear at first read this was a list of files vs. names or raw yaml.